runtime/tests/tests.rs

```diff
@@ -618,6 +622,76 @@ fn generic_asset_transfer_works_with_fee_exchange() {
 		});
 }
 
+#[test]
+fn generic_asset_transfer_works_with_doughnut() {
+    let method_obj = cennznut::Method {
```

Comment:
[suggestion] Can use `make_runtime_cennznut` from the `doughnut.rs` (will need to make the helper public, but that should be fine since it is just a test helper anyway)